sr_description_common/models/bare_metal_table/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>bare_metal_table</name>
```

Comment:
is this the really small table under the arm? maybe we should rename it to something more explicit